build.rs

```diff
@@ -21,11 +20,14 @@ fn get_path_from_llvm(p: &str, s: &[&str]) -> PathBuf {
 }
 
 fn main() {
-    // Tell cargo to tell rustc to link the system bzip2
+    // Tell cargo to tell rustc to link the system flux-core
     // shared library.
     println!("cargo:rustc-link-lib=flux-core");
 
-    if let Ok(llvm_config_path) = which("llvm-config") {
+    let libclang_path = env::var("LIBCLANG_PATH");
+    if libclang_path.is_ok() {
```

Comment:
This is stylistic, but since the else is an `if let` I would probably make this one too:

```suggestion
    if let libclang_path = env::var("LIBCLANG_PATH") {
```

Reply:
Oooo. I didn't realize you could do that.  Very nice! just pushed a fix.